### PR TITLE
Build for Ledger Nano X

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose

To add support for building the app with the Nano X Secure SDK, so that we can test and verify that the application also works on a Nano X device.

## Changes
- Added the Nano X Secure SDK as a submodule dependency.
- Changed the pagination UI component from `bn_paging` to `bnnn_paging` as the first is not known by the Nano X SDK.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings. (Note: the SDK emits some warnings, but we cannot fix those)
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,17 +43,31 @@ device, make sure to deploy a custom certificate to the device. See the "PIN Byp
 For documentation of the exposed functionality and how to integrate with the Concordium specific 
 applications, please take a look [here](doc/api.md).
 
+## Building for the Speculos emulator
```
Comment (shjortConcordium) [inline README.md:46]: We should add, somewhere, that you need _specifically_ clang-9 to build this.
(And that the "ld.lld doesn't exist" error is due to this)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
 # Concordium Ledger application
```
Comment (shjortConcordium) [inline README.md:1]: We should add, somewhere, that you have to run `make clean` when switching which SDK is used.

File: README.md (hunk 1 above)
Comment (orhoj) [reply README.md:57]: I have added it to the readme.

File: README.md (hunk 2 above)
Comment (orhoj) [reply README.md:1]: I have added it to the readme.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**